package.json

```diff
@@ -77,5 +77,10 @@
     "build-and-test:local": "yarn build && yarn package:validate && yarn lint && yarn format:check && yarn test && yarn typecheck",
     "package:validate": "zx scripts/package.validate.mjs"
   },
-  "packageManager": "yarn@3.6.3"
+  "packageManager": "yarn@3.6.3",
+  "dependencies": {
+    "mobx": "^6.12.0",
+    "mobx-react-lite": "^4.0.5",
+    "mobx-state-tree": "^5.4.0"
+  }
```

Comment:
Whoops, I should have done these in the sub-package. Although maybe that's why this works and the original one didn't? I'll try and move it and find out.

Reply:
This is the root of the problems Jamon and I were seeing earlier today. 

When I ran:

1. `yarn remove mobx mobx-react-lite mobx-state-tree`
2. `cd apps/reactotron-app`
3. `yarn add mobx mobx-react-lite-mobx-state-tree`
4. `cd ../..
5. `yarn build`
6. `yarn start`

I got this error, as before:

```
/Users/tylerwilliams/reactotron/node_modules/react/cjs/react.development.js:209 Warning: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://reactjs.org/link/invalid-hook-call for tips about how to debug and fix this problem.
```

<img width="710" alt="Screenshot 2023-12-16 at 10 31 26 PM" src="https://github.com/coolsoftwaretyler/reactotron/assets/29899554/23a001ff-2df5-4736-8fce-725c203c3d48">

I'm not sure exactly why this matters, but something about the yarn workspace, the nx build steps, or the webpack configs doesn't like seeing mobx and mobx-react-lite and mobx-state-tree in the nested directory is a problem, but at the root, things work fine.

`mobx-react-lite` only has React as a peer dependency, but maybe that's causing a problem somewhere?

https://github.com/mobxjs/mobx/blob/main/packages/mobx-react-lite/package.json

Reply:
I left this PR in a working state for comparison since Jamon has a non-working branch. Haven't merged the broken changes, but they should be easy to reproduce.